### DISCUSSION TITLE
Welcome to bisecur2mqtt Discussions!

### DISCUSSION THREAD

**Discussion by petrvlcek** [petrvlcek/bisecur2mqtt#1] [General]
## 👋 Welcome!
  This repository does not have documentation yet. If you want to try it out and would like to know more, just ask questions here instead of creating issues.

**Comment by blubkens** [comment]
Hi petrvlcek,

I just read this after creating an issue and pull.
Hope that it isn't a problem?
Thanks,

Blubkens.
